Python/marshal.c

```diff
@@ -875,18 +872,16 @@ r_PyLong(RFILE *p)
         }
         d += (digit)md << j*PyLong_MARSHAL_SHIFT;
     }
-    if (PyErr_Occurred()) {
-        Py_DECREF(ob);
-        return NULL;
-    }
```

Comment:
Perhaps leave an assert:

```suggestion
    assert(!PyErr_Occurred());
```

Reply:
Applying suggestions on deleted lines is currently not supported. :-(

Reply:
That's strange! The "Commit suggestion" button seems to be enabled in my browser.

Reply:
Could you please commit it?

Reply:
> That's strange! The "Commit suggestion" button seems to be enabled in my browser.

Moreover, it allows to enter a commit message but fails on the next step:

![image](https://github.com/python/cpython/assets/4881073/6635b777-85db-4eb8-a3b0-c639eb3248a3)

Reply:
The editor opened from the PR files view (![the ellipsis button](https://github.com/python/cpython/assets/4881073/f4cd7f33-0352-46bb-b81a-80e95414fb47) → Edit file) seems to work better for such changes.

Reply:
> Could you please commit it?

It seems @arhadthedev already did so.

Reply:
Thanks, @arhadthedev.